src/styles/links/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Links
+description: Use links to navigate between pages
+section: Styles
+layout: layout-pane.njk
+showPageNav: true
+---
+
+{% from "_example.njk" import example %}
+
+Links are blue and underlined by default. If your link is at the end of a sentence or paragraph, make sure that the linked text does not include the full stop.
+
+{{ example({group: "styles", item: "links", example: "default", html: true, open: true}) }}
+
+## Links without a visited state
+
+Use the `govuk-link--no-visited-state` modifier class where it is not helpful to distinguish between visited and unvisited states, for example when linking to pages with frequently-changing content such as the dashboard for an admin interface.
+
+{{ example({group: "styles", item: "links", example: "no-visited-state", html: true, open: true}) }}
```

Comment:
Super minor, but can you check this page copies in the latest content?

This line had some formatting on it recently so just double checking

```suggestion
{{ example({ group: "styles", item: "links", example: "no-visited-state", html: true, open: true }) }}
```